### PR TITLE
Added inital sping rest configuration

### PR DESCRIPTION
**Pull request by makro121**

### CODE REVIEW

File: backend/pom.xml [hunk 1]
```diff
@@ -15,4 +32,13 @@
         <maven.compiler.target>1.7</maven.compiler.target>
```
Comment (piotrjanuszek) [inline backend/pom.xml]: co to jes?
bo jak wersja javy to trochę stara

File: backend/src/main/java/com/makro/model/SimpleTime.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        return this;
+    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public float getTime() {
+        return time;
+    }
```
Comment (piotrjanuszek) [inline backend/src/main/java/com/makro/model/SimpleTime.java]: ja to bym lomboka zainstalował do tego

File: backend/src/main/java/com/makro/model/SimpleTime.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+    public SimpleTime withUsername(String username) {
+        this.username = username;
+        return this;
+    }
+
+    public SimpleTime withTime(float time) {
+        this.time = time;
+        return this;
+    }
```
Comment (piotrjanuszek) [inline backend/src/main/java/com/makro/model/SimpleTime.java]: po co się to robi?

File: backend/pom.xml (hunk 1 above)
Comment (makro121) [reply backend/pom.xml]: Yeah, could be changed to newer one

File: backend/src/main/java/com/makro/model/SimpleTime.java (hunk 2 above)
Comment (makro121) [reply backend/src/main/java/com/makro/model/SimpleTime.java]: What is lombok? This is only poc version, I think that we can think about improvements in following PRs.

File: backend/src/main/java/com/makro/model/SimpleTime.java (hunk 3 above)
Comment (makro121) [reply backend/src/main/java/com/makro/model/SimpleTime.java]: Right now we can write: new SimpleTime().withUsername("Julia").withTime(2.15);

File: backend/pom.xml (hunk 1 above)
Comment (makro121) [reply backend/pom.xml]: Actually, it was copy pasted from maven template, i think we can remove that but need to confirm

File: backend/src/main/java/com/makro/model/SimpleTime.java (hunk 2 above)
Comment (piotrjanuszek) [reply backend/src/main/java/com/makro/model/SimpleTime.java]: https://projectlombok.org/

**Pull request merged**